manager/dispatcher/assignments.go

```diff
@@ -245,3 +250,29 @@ func (a *assignmentSet) message() api.AssignmentsMessage {
 
 	return message
 }
+
+// secret populates the secret value from raft store. For external secrets, the value is populated
+// from the secret driver.
+func (a *assignmentSet) secret(secretID string, readTx store.ReadTx) (*api.Secret, error) {
+	secret := store.GetSecret(readTx, secretID)
```

Comment:
Thanks @cpuguy83, as far as I understand no, for two reasons:
1. We need to fetch the actual `Driver` object in case additional `Driver` metadata is needed for initiating the plugin
2. We need the `api.Secret` to initiate the assignment. Since I have the secret ID, name and value I can probably generate this object. However, I feel that using the raft store is more robust. WDYT?